app/models/user.rb

```diff
@@ -10,45 +10,49 @@ class User < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
 
-  has_many :sent_friendships, class_name: 'Friendship', foreign_key: 'sender_id'
-  has_many :received_friendships, class_name: 'Friendship', foreign_key: 'receiver_id'
+  has_many :friendships, class_name: 'Friendship', foreign_key: 'user_id'
+  has_many :received_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
 
   def friends
-    sent_friendship_requests = sent_friendships.map { |friendship| friendship.receiver if friendship.accepted }
-    received_friendship_requests = received_friendships.map { |friendship| friendship.sender if friendship.accepted }
-    (sent_friendship_requests + received_friendship_requests).compact
+    friends = friendships.map { |friendship| friendship.friend if friendship.accepted }
```

Comment:
- [ ] `friends` or `pending_friends` can be defined as association with SQL query instead of processing objects in application memory (as the database was created to store and query data it is always more efficient than ruby)

This can be done with:
```ruby
has_many :friends, -> { where(friendships: { accepted: true }) }, through: :friendships
```
Same for `pending_friends`